installing/installing_ibm_cloud_public/configuring-iam-ibm-cloud.adoc

```diff
@@ -0,0 +1,24 @@
+[id="configuring-iam-ibm-cloud"]
+= Configuring IAM for IBM Cloud
+include::modules/common-attributes.adoc[]
+:context: configuring-iam-ibm-cloud
+
+toc::[]
+
+In environments where the cloud identity and access management (IAM) APIs are not reachable, you must put the Cloud Credential Operator (CCO) into manual mode before you install the cluster.
+
+include::modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
[role="_additional-resources"]
[id="additional-resources_configuring-iam-ibm-cloud"]
== Additional resources

per updated guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Confirmed with Katherine earlier that .Additional resources is still OK, but the role and anchor ID is requiered.

Reply:
Done.

Reply:
Done.

Reply:
@ahardin-rh So based on the conversation in the peer review Slack channel, @kalexand-rh confirmed that using .`Additional resources` below a module is still legit.